docs/API/REST-V2-Usage.mdx

```diff
@@ -4,6 +4,9 @@ hide_table_of_contents: true
 
 # REST v2 API
 
+To use Evergreen's REST v2 API, all URLs should be prefixed with `https://evergreen.mongodb.com/rest/v2`. For
+example, to [get a single task](#tag/tasks/paths/~1tasks~1{task_id}/get), make a GET request to `https://evergreen.mongodb.com/rest/v2/tasks/<TASK_ID>`.
```

Comment:
should we use `https://evergreen.mongodb.com/rest/v2/tasks/{task_id}` to match the format lower down on the page?